content/en/contribute/code/repository-checklist.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Repository Checklist"
+linkTitle: "Repository Checklist"
+weight: 5
+description: >
+  Checklist to consider when creating repositories under Medic's GitHub account
+---
+
+## Repository Checklist
+When creating a new repository under [Medic's GitHub organization](https://github.com/medic), the contributor(s) should at least do the following:
+
+### Source Control
+- [ ] The `main` branch is locked via [branch protection rules](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule).
+- [ ] Merges are done through PRs.
+- [ ] Issue templates exist.
+- [ ] PR template exists.
+- [ ] PRs reference related issues.
+- [ ] Commit formats follow the [guidelines]({{< ref "workflow/#commits" >}}). 
+- [ ] Secrets are not part of the commit history or made public.
+- [ ] For public repositories the `main` branch should contain the following files:
+    - `LICENSE` ([example](https://github.com/medic/cht-core/blob/master/LICENSE))
+    - `README.md`
+
+### Testing
+- [ ] Unit tests cover the majority of the code (>90% if possible).
```

Comment:
We've discussed internally and haven't agreed on coverage targets. This is probably a separate conversation. To keep this change uncontroversial, let's drop it.

```suggestion
- [ ] Unit tests cover the majority of the code.
```